Library/Formula/bash-completion.rb

```diff
@@ -32,8 +33,15 @@ def install
 
     system "./configure", "--prefix=#{prefix}"
     system "make", "install"
+  end
 
-    unless (compdir/"brew_bash_completion.sh").exist?
+  def post_install
+    # This symlink needs to be created in the post-install step. Otherwise, a
+    # copy of our official bash completion for `brew` will end up in the bottle,
+    # which is not very helpful in keeping this up-to-date as Homebrew updates.
+    brew_comp = compdir/"brew_bash_completion.sh"
+    unless brew_comp.symlink?
```

Comment:
Yes, normally the rule would be not to touch anything that already exists in `etc`. Ideally, I would make sure the `unlink` below this line doesn't remove a file that is no longer identical to the one from the bottle (e.g., has been edited by the user) but I think this is unlikely enough to be the case, so the complication a more thorough check would bring isn't warranted.